src/components/Duration.tsx

```diff
@@ -2,20 +2,25 @@ import React, { useState } from 'react'
 
 import { usePrestoEnabledStateClass, usePrestoUiEvent } from '../react'
 import {
-  BasePlayerComponentProps,
+  BaseComponentProps,
   getMinimalFormat,
   timeToString,
 } from '../utils'
 
 import { Label } from './Label'
 
-export type DurationProps = BasePlayerComponentProps
+export interface DurationProps extends BaseComponentProps {
+  children?: React.ReactNode
+}
 
+/**
+ * Duration.
```

Comment:
Do we want to add here more details? Or I am maybe missing point of this description?
There are few other similar descriptions. Maybe we can followup on these in documentation task.

Reply:
Totally agree, my problem is that I actually don't know yet what every component does :D, once I find out, I'll improve the descriptions.